### ISSUE TITLE
Forbidding method calls of Object.create(null)

### ISSUE THREAD

**Issue by chicoxyzzy** [facebook/flow#3177]
https://flowtype.org/try/#0PTAEAEDMBsHsHcBQiDGsB2BnALqAhgFygDeAvqALygDyARgFYCmK2AdCgE6N7aMAU6AK7RoASgDciPK2ywAytg4BLdAHM+E0KBChMAC1jCAJqFqNQSkY1V5oU1nGiVQAckcvx2sPsPQTZ0GhrW0QgA

```js
// @flow

const a: {} = Object.create(null);
a.toString();  // should be illegal
a.lol = 'lol'; // should be legal

```

I know there is an [issue about assigning properties on object with null prototype](https://github.com/facebook/flow/issues/813#issuecomment-175458604) but didn't find any about forbidding calls of Object.prototype methods.

**Comment by danielo515** [comment]
I am suffering this problem too, and some other problems derived from the same situation.

**Comment by davidtheclark** [comment]
This does not seem to be fixed in v0.56. 

The following should error but does not:

```js
/* @flow */
const a = Object.create(null);
a.toString();
```

See https://flow.org/try/#0PQKgBAAgZgNg9gdzCYAoVBjOA7AzgFzAEMwBeMAeQCMArAUw3wDoMAnOo-OgCmwFcYMAJQBuVESb44AZXysAltgDm3UUA
